hw/top_earlgrey/dv/vendor_chip_sim_cfg_sample.hjson

```diff
@@ -0,0 +1,15 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+{
+  // Import additional common sim cfg files.
+  import_cfgs: ["{proj_root}/hw/top_earlgrey/dv/chip_sim_cfg.hjson"]
```

Comment:
By importing the existing `chip_sim_cfg.hjson`, absolutely nothing needs to be replicated - you can simply run all existing tests at the chip level with a modified design or dv.

Reply:
thanks for this change @sriyerg 
Just to clarify though, even with this change, if the search order of the `cores-root` directory are different from the command line order, we could potentially still have problems right? That's probably something I can try out.